test/database.spec.js

```diff
@@ -356,6 +356,51 @@ describe('database', () => {
                 });
             });
         });
+
+        describe('checkpoint', () => {
+            it('should create checkpoint', async () => {
+                const pairs = [
+                    { key: getRandomBytes(), value: getRandomBytes() },
+                    { key: getRandomBytes(), value: getRandomBytes() },
+                ];
+                const batch = new Batch();
+                for (const kv of pairs) {
+                    batch.set(kv.key, kv.value);
+                }
+                await db.write(batch);
+
+                await expect(db.has(pairs[0].key)).resolves.toEqual(true);
+                await expect(db.has(pairs[1].key)).resolves.toEqual(true);
+
+                const tmpPath = fs.mkdtempSync(os.tmpdir() + '/');
```

Comment:
I think you don't need the input to this file? also, probably it's better to do like below so that even if it fails in the middle, it's always cleaned up
```
let tmpPath;
beforeEach(() => {
  tmpPath = fs.mkdtempSync();
});
afterEach(() => {
  fs.rmSync(tmpPath, { recursive: true, force: true });
});
```

Reply:
Thank you for your suggestions. Done.